pyvista/core/filters.py

```diff
@@ -2634,6 +2634,38 @@ def shrink(dataset, shrink_factor=1.0, progress_bar=False):
         if isinstance(dataset, vtk.vtkPolyData):
             return output.extract_surface()
 
+    def reflect(dataset, plane, copy=False, center=0):
+        """Reflect a dataset across a plane.
+
+        Parameters
+        ----------
+        plane : str
+            Reflection plane options: `'xmin'`, `'ymin'`, `'zmin'`, `'xmax'`, `'ymax'`, `'zmax'`,
+            `'x'`, `'y'`, or `'z'`.
```

Comment:
```suggestion
            Reflection plane options: ``'xmin'``, ``'ymin'``,
            ``'zmin'``, ``'xmax'``, ``'ymax'``, ``'zmax'``, ``'x'``,
            ``'y'``, or ``'z'``.
```
Our docs are generated by sphinx, which follows the rst format which requires two "`" when declaring a literal.

Reply:
Ok! Thanks!